src/jit/importer.cpp

```diff
@@ -3850,6 +3759,18 @@ GenTreePtr Compiler::impIntrinsic(GenTreePtr            newobjThis,
                 break;
             }
 
+            case NI_MathF_Round:
+            case NI_Math_Round:
+            {
+                // Math.Round and MathF.Round used to be a legacy JIT intrinsic. In order
+                // to simplify the transition, we will just treat it as if it was still the
```

Comment:
Nitpick: "legacy" has special meaning in the jit code base. Maybe say "used to be a traditional jit intrinsic."

Reply:
Fixed to use 'traditional' rather than legacy.